velox/docs/velox-in-10-min.rst

```diff
@@ -25,6 +25,15 @@ The code used in this tutorial is available in `velox/exec/tests/VeloxIn10MinDem
 
 Let’s get started.
 
+Before running code examples below, you'll need to run a bit of boilerplate first:
```

Comment:
This line here : 
``` 
The code used in this tutorial is available in `velox/exec/tests/VeloxIn10MinDemo.cpp <https://github.com/facebookincubator/velox/blob/main/velox/exec/tests/VeloxIn10MinDemo.cpp>`_.
``` 
links to the example code and adds the type resolver and registers the scalar functions . Do we want this to just to bring it to folks attention ? If so it would be nice to add more detail on what these functions actually do.